.github/workflows/docs_deployment.yaml

```diff
@@ -41,13 +41,14 @@ jobs:
           python-version: 3.11
           architecture: x64
           cache: 'pip'
-
       - name: Install libboost
         run: sudo apt install libboost-dev
```

Comment:
Is libboost not installed by `install_zutils`?

Reply:
Running skip_apt since everything else exists in the docker image.